core/src/components/menu/menu.tsx

```diff
@@ -619,12 +620,17 @@ export class Menu implements ComponentInterface, MenuI {
       // emit open event
       this.ionDidOpen.emit();
 
-      // focus menu content for screen readers
-      if (this.menuInnerEl) {
-        this.focusFirstDescendant();
+      /**
+       * Move focus to the menu to prepare focus trapping, as long as
+       * it isn't already focused. Use the host element instead of the
+       * first descendant to avoid the scroll position jumping around.
+       */
+      const focusedMenu = document.activeElement?.closest('ion-menu');
+      if(!focusedMenu || focusedMenu !== this.el) {
```

Comment:
Optional: This can be simplified:

```suggestion
      if (focusedMenu !== this.el) {
```

If `focusedMenu` is `null` (possible return value from closest()) then it won't be equal to `this.el` which is an HTML element.